src/bridge/SequencerInbox.sol

```diff
@@ -450,8 +450,6 @@ contract SequencerInbox is
 
     delayProofImpl(afterDelayedMessagesRead, delayProof);
 
-    emit LastHotshotHeight(sequenceNumber, delayProof.hotshotHeight);
```

Comment:
Correct me if I'm wrong @Sneh1999, but won't we need the hotshot height in all of the add batch methods for the timeboost integration correct?